Add course description for RSE Leadership Training. (#792)
open-life-science/open-life-science.github.io: _events/2024-10-research-software-engineer-training-course.md

---
layout: event
title: Research Software Engineer (RSE) Leadership Training Course
type: workshop
description: |
    This leadership course is possibly the first in the world to provide an intersectional framework that underpins a leadership course for those working as RSEs and those in  the tech industry.

contributions:
  organisers:
  - rowland-mosbergen

date_start: 2024-10-17
date_end: 2024-11-28

location:
  name: Online

contact_email: rowland@we-are-ols.org

registration:
  link: 
  deadline: 
---

* **Course description:**
  * This leadership course is possibly the first in the world to provide an **intersectional framework** that underpins a leadership course for those working as RSEs and those in the tech industry.
  * It is also unique in that we meet with you first to understand what challenges you are facing and to to create an individual development plan just for you.
  * We will then craft a practical training course just for you and a small group of other leaders to ensure all your needs are being met as an individual and as a group.
  * Course will run weekly from **17th of October** to **28th of November.**

> * **Topics covered include:**

Author: ```suggestion
* **Workshop topics:**
```

  * Using a Diversity and Inclusion framework to improve your team.
  * Using Continuous Improvement skills to identify talent.
  * Setup and maintain pipelines to attract talent.
  * Project Management in complex scenarios.
  * Managing high-performance teams.
  * How to deal with difficult change management situations and build relationships.

* **Pricing:** 
  * **£499 GBP** for initial skills consult and personal development plan.

>   * **£1999 GBP** ffor the practical training course designed just for you and a small group of other leaders.

Author: ```suggestion
  * **£1999 GBP** for a series of workshops with a small number of other leaders, that is crafted to your needs based on the initial consultation.
```

contributions:
  organisers:
  - rowland-mosbergen

date_start: 2024-10-17
date_end: 2024-11-28

location:
  name: Online

contact_email: rowland@we-are-ols.org

registration:
  link: 
  deadline: 
---

# Course description

This leadership course is possibly the first in the world to provide an **intersectional framework** that underpins a leadership course for those working as RSEs and those in the tech industry.

It is also unique in that we meet with you first to understand what challenges you are facing and to to create an individual development plan just for you.

We will then craft a practical training course just for you and a small group of other leaders to ensure all your needs are being met as an individual and as a group.

# Topics

The course will cover
* Using a Diversity and Inclusion framework to improve your team.
* Using Continuous Improvement skills to identify talent.
* Setup and maintain pipelines to attract talent.
* Project Management in complex scenarios.
* Managing high-performance teams.
* How to deal with difficult change management situations and build relationships.

# Organization

> Course will run weekly from **17th of October** to **28th of November.**

Reviewer: When will the course be during the weeks? How long will be the sessions?

---
layout: event
title: Research Software Engineer (RSE) Leadership Training Course
type: workshop
description: |
    This leadership course is possibly the first in the world to provide an intersectional framework that underpins a leadership course for those working as RSEs and those in  the tech industry.

contributions:
  organisers:
  - rowland-mosbergen

date_start: 2024-10-17
date_end: 2024-11-28
duration: "02:45:00"

location:
  name: Online

contact_email: rowland@we-are-ols.org

registration:

>   link: 
>   deadline: 

Reviewer: ```suggestion
  link: https://events.humanitix.com/rse-leadership-course
  deadline: 2024-09-24
```

---

# Course description

This leadership course is possibly the first in the world to provide an **intersectional framework** that underpins a leadership course for those working as RSEs and those in the tech industry.

It is also unique in that we meet with you first to understand what challenges you are facing and to to create an individual development plan just for you.

We will then craft a practical training course just for you and a small group of other leaders to ensure all your needs are being met as an individual and as a group.

# Topics

The course will cover
* Using a Diversity and Inclusion framework to improve your team.
* Using Continuous Improvement skills to identify talent.
* Setup and maintain pipelines to attract talent.
* Project Management in complex scenarios.
* Managing high-performance teams.
* How to deal with difficult change management situations and build relationships.

> # Organization

Reviewer: ```suggestion
# Sessions
```

> Course will run weekly from **17th of October** to **28th of November.**

Author: ```suggestion
Course will run weekly from **17th of October** to **28th of November, on Tuesdays or Fridays, based on participant availability.**
```

> Course will run weekly from **17th of October** to **28th of November, on Tuesdays or Fridays, based on participant availability.**

Reviewer: ```suggestion
- **Dates**: from **17th of October** to **28th of November
- **Frequency**: weekly, on Tuesdays or Fridays, based on participant availability
- **Duration**: around two and a half hours
```

---

# Course description

This leadership course is possibly the first in the world to provide an **intersectional framework** that underpins a leadership course for those working as RSEs and those in the tech industry.

It is also unique in that we meet with you first to understand what challenges you are facing and to to create an individual development plan just for you.

We will then craft a practical training course just for you and a small group of other leaders to ensure all your needs are being met as an individual and as a group.

# Topics

The course will cover
* Using a Diversity and Inclusion framework to improve your team.
* Using Continuous Improvement skills to identify talent.
* Setup and maintain pipelines to attract talent.
* Project Management in complex scenarios.
* Managing high-performance teams.
* How to deal with difficult change management situations and build relationships.

# Sessions

- **Dates**: from **17th of October** to **28th of November
- **Frequency**: weekly, on Tuesdays or Fridays, based on participant availability
- **Duration**: around two and a half hours

# Pricing

* **£499 GBP** for initial skills consult and personal development plan.

> * **£1999 GBP** for a series of workshops with a small number of other leaders, that is crafted to your needs based on the initial consultation.

Reviewer: Is this pricing for the sessions later this year?

Reviewer 2: Yes, the 1999 is for the workshops that are to happen on Tuesdays or Fridays.
